care/apps/patients/fixtures/patients_fixture_dummy.csv

```diff
@@ -0,0 +1,501 @@
+id,icmr_id,govt_id,name,gender,year,month,phone_number,facility_id,nearest_facility_id,date_of_birth,blood_group,created_by,address,district,covid_status,clinical_status,portea_able_to_connect,patient_status,created_at,updated_at

```

Comment:
Similar to Facility & Testing Labs. Update Patients Model for Address and use here.